src/VisualStudio/Core/Def/Implementation/Remote/JsonRpcClient.cs

```diff
@@ -161,6 +157,10 @@ private bool ReportUnlessCanceled(Exception ex, CancellationToken cancellationTo
                 return true;
             }
 
+            // once we fix all pending vso bugs, we should remove these 
```

Comment:
Can you clarify "pending VSO bugs" to make it clear this is relating to StreamJsonRpc or something? I presume you don't mean we fix _every_ bug in VSO.